app/assets/javascripts/gallery.js.erb

```diff
@@ -690,8 +677,8 @@ $(document).ready(function(){
       success: function(){
         window.location.replace('/admin/warning');
       },
-      failure: function(response){
-        bootbox.alert(response.statusText);
+      error: function(response){
```

Comment:
Why did you change this one to error, but line 704 is still failure?

Reply:
Good catch. Will be fixed in new commit (for all JS files). All should be "error" for consistency.